src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -68,6 +70,18 @@ trait IterableOps[+A] extends Any {
   /** The first element of the collection. */
   def head: A = iterator().next()
 
+  /** Selects the last element.
+    * $orderDependent
+    * @return The last element of this $coll.
+    * @throws NoSuchElementException If the $coll is empty.
+    */
+  def last: A = {
+    var lst = head
```

Comment:
This is very unlikely to be efficient since you create an iterator in addition to going through a foreach, which by default will create a second iterator.

Just do this manually:

```scala
val i = iterator()
var lst = i.next()
while (i.hasNext) lst = i.next()
lst
```